### PR TITLE
Add login page

### PR DESCRIPTION
**Pull request by zackharley**
### Includes
* Login Page
    * Uses Semantic UI classes wrapping Redux Form components for easy integration with Redux stores

### Screenshots

*Mobile*

localhost_8081_login galaxy s5

*Tablet*

localhost_8081_login ipad

*Laptop*

screen shot 2017-11-15 at 1 33 19 pm

### CODE REVIEW

File: client/components/Login/Login.js [hunk 1]
```diff
@@ -1,51 +1,77 @@
...
+                                    size="huge"
+                            >
+                                Dream it. Build it.
+                            </Header>
+                            <Header as="h2"
+                                    className="fontWeight-lighter"
+                                    inverted
+                                    size="medium"
+                            >
+                                QHacks 2018, Queen's University
```
Comment (RobertWSaunders) [inline client/components/Login/Login.js]: Escape the apostrophe here.

File: client/components/utils/LoginForm.js [hunk 2]
```diff
@@ -1,23 +1,62 @@
...
+                </Button>
+            </Form>
+            <div className="fontSize-medium"
+                 style={{ marginTop: '40px' }}
+            >
+                <Link to="/reset-password">Forgot password?</Link>
+            </div>
+            <Divider/>
+            <p className="fontSize-large">
+                Haven't applied yet? <Link to="/apply">Apply Here</Link>
```
Comment (RobertWSaunders) [inline client/components/utils/LoginForm.js]: Escape apostrophe.

File: client/components/Login/Login.js (hunk 1 above)
Comment (zackharley) [reply client/components/Login/Login.js]: No need to do that. That is just a syntax highlighting issue ([as seen in Atom](https://github.com/orktes/atom-react/issues/3))

File: client/components/utils/LoginForm.js (hunk 2 above)
Comment (zackharley) [reply client/components/utils/LoginForm.js]: No need to do that. That is just a syntax highlighting issue ([as seen in Atom](https://github.com/orktes/atom-react/issues/3))

File: client/components/Login/Login.js (hunk 1 above)
Comment (RobertWSaunders) [reply client/components/Login/Login.js]: Fair enough, good catch, nice reference in issue too.

File: client/components/utils/LoginForm.js (hunk 2 above)
Comment (RobertWSaunders) [reply client/components/utils/LoginForm.js]: Fair enough, good catch, nice reference in issue too.

**Pull request merged**